android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -546,6 +558,8 @@ public void connectToRoom() {
 
         connectOptionsBuilder.enableDominantSpeaker(this.dominantSpeakerEnabled);
 
+        connectOptionsBuilder.encodingParameters(new EncodingParameters(16, this.videoDimensions.width <= 640 ? 512 : 0));
```

Comment:
can you add the comment from original ionic pr

Reply:
added